Library/DiscUtils.Core/Vfs/VfsFileSystem.cs

```diff
@@ -707,8 +707,11 @@ private void DoSearch(List<string> results, string path, Regex regex, bool subFo
         private TDirEntry ResolveSymlink(TDirEntry entry, string path)
         {
             TDirEntry currentEntry = entry;
+            if (path.Length > 0 && path[0] != '\\')
+            {
+                path = '\\' + path;
+            }
```

Comment:
In principle we have `Path.IsRooted` - but we need to consider OS-specifics here.